psycopg2cffi/__init__.py

```diff
@@ -41,7 +41,7 @@ def _param_escape(s,
 
 def connect(dsn=None,
         database=None, user=None, password=None, host=None, port=None,
-        connection_factory=None, cursor_factory=None, async=False, **kwargs):
```

Comment:
It will be better if ``async_`` is present as a keyword in the function signature: it's functionally equivalent but it's more friendly to IDEs and people reading the code.

Reply:
When I added `async_` as a keyword in the function, tests haven't been passed so it isn't functionally equivalent ([commit](https://github.com/chtd/psycopg2cffi/pull/92/commits/772fe9244c9006b4255374f7fa1dc5ab56dba127), [Travis](https://travis-ci.org/chtd/psycopg2cffi/builds/304747230?utm_source=github_status&utm_medium=notification))